recipes/antlr4-python3-runtime/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "antlr4-python3-runtime" %}
+{% set version = "4.9.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/antlr4-python3-runtime-{{ version }}.tar.gz
+  sha256: 31f5abdc7faf16a1a6e9bf2eb31565d004359b821b09944436a34361929ae85a
+
+build:
+  number: 0
```

If we limit this to `python>=3.6` (that's the range we support on conda-forge only), this can be `noarch: python`. This will make the things a lot simpler.

```suggestion
  number: 0
  noarch: python
```